__test__/getProductById.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require('supertest');
+const app = require('../server/app');
+const connection = require('../server/database/connection');
+const buildDb = require('../server/database/data/build');
+const { getProductById } = require('../server/database/queries');
+
+describe('Test Routes, Test Database getAllProducts query', () => {
```

Comment:
please change the description 
```suggestion
describe(`Test '/product/:productId' route, and test  get product by id  query`, () => {
```

Reply:
done